lms/envs/bok_choy.py

```diff
@@ -190,44 +108,7 @@
 ############################# SECURITY SETTINGS ################################
 # Default to advanced security in common.py, so tests can reset here to use
 # a simpler security model
-FEATURES['ENABLE_MAX_FAILED_LOGIN_ATTEMPTS'] = False
-FEATURES['SQUELCH_PII_IN_LOGS'] = False
-FEATURES['PREVENT_CONCURRENT_LOGINS'] = False
-
-FEATURES['ENABLE_MOBILE_REST_API'] = True  # Show video bumper in LMS
-FEATURES['ENABLE_VIDEO_BUMPER'] = True  # Show video bumper in LMS
-FEATURES['SHOW_BUMPER_PERIODICITY'] = 1
-
-# Enable courseware search for tests
-FEATURES['ENABLE_COURSEWARE_SEARCH'] = True
-
-# Enable dashboard search for tests
-FEATURES['ENABLE_DASHBOARD_SEARCH'] = True
-
-# discussion home panel, which includes a subscription on/off setting for discussion digest emails.
-FEATURES['ENABLE_DISCUSSION_HOME_PANEL'] = True
-
-# Enable support for OpenBadges accomplishments
-FEATURES['ENABLE_OPENBADGES'] = True
 
-FEATURES['ENABLE_LTI_PROVIDER'] = True
-INSTALLED_APPS.append('lti_provider.apps.LtiProviderConfig')
-AUTHENTICATION_BACKENDS.append('lti_provider.users.LtiBackend')
```

Comment:
These two removals that append to `INSTALLED_APPS` and `AUTHENTICATION_BACKENDS` don't seem to have a corresponding addition—were they superfluous?

Reply:
Yes, if the feature is enabled in the config file, then production.py will add these already.

https://github.com/edx/edx-platform/blob/master/lms/envs/production.py#L754-L756

Reply:
Ah, great!